src/renderer/windows/main/components/Snippets.vue

```diff
@@ -82,29 +83,54 @@
                             v-model="editing.key"
                             v-focus
                         >
-                        <label
-                            tabindex="0"
-                            for="regex"
-                            @keydown.space="editing.regex = !editing.regex"
-                            class="flex justify-center items-center cursor-pointer group"
-                        >
-                            <input type="checkbox" id="regex" class="invisible" v-model="editing.regex">
-                            <div class="mr-2">
-                                <icon-checkbox
-                                    v-if="editing.regex"
-                                    :checked="true"
-                                    class="text-brand-blue fill-current w-6 h-6 rounded-sm group-focus:shadow-outline"
-                                ></icon-checkbox>
-                                <icon-checkbox
-                                    v-else
-                                    :checked="false"
-                                    class="fill-current w-6 h-6 rounded-sm group-focus:shadow-outline"
-                                    :class="['text-grey-lightest', 'text-grey-darkest'][theme]"
-                                ></icon-checkbox>
-                            </div>
+                        <div class="flex flex-col items-start">
+                            <label
+                                tabindex="0"
+                                for="active"
+                                @keydown.space="editing.active = !editing.active"
+                                class="flex justify-center items-center cursor-pointer group"
+                            >
+                                <input type="checkbox" id="active" class="invisible" v-model="editing.active">
+                                <div class="mr-2">
+                                    <icon-checkbox
+                                        v-if="editing.active"
+                                        :checked="true"
+                                        class="text-brand-blue fill-current w-6 h-6 rounded-sm group-focus:shadow-outline"
+                                    ></icon-checkbox>
+                                    <icon-checkbox
+                                        v-else
+                                        :checked="false"
+                                        class="fill-current w-6 h-6 rounded-sm group-focus:shadow-outline"
+                                        :class="['text-grey-lightest', 'text-grey-darkest'][theme]"
+                                    ></icon-checkbox>
+                                </div>
 
-                            Use Regular Expression
-                        </label>
+                                active
```

Comment:
"active" -> "Active" to preserve Title Case style.

Reply:
good catch